src/components/organisms/ChatSidebar/ChatSidebar.tsx

```diff
@@ -53,50 +43,41 @@ export const ChatSidebar: React.FC = () => {
   });
 
   return (
-    <>
-      <div className={containerStyles}>
-        <div className="chat-sidebar__header">
-          <div className="chat-sidebar__controller" onClick={toggleSidebar}>
-            {isExpanded ? (
-              <FontAwesomeIcon icon={faChevronRight} size="sm" />
-            ) : (
-              <>
-                <FontAwesomeIcon icon={faChevronLeft} size="sm" />
-                <FontAwesomeIcon
-                  className="chat-sidebar__controller__second-icon"
-                  icon={faCommentDots}
-                  size="lg"
-                />
-              </>
-            )}
-          </div>
+    <div className={containerStyles}>
+      <div className="chat-sidebar__header">
+        <div className="chat-sidebar__controller" onClick={toggleSidebar}>
+          {isExpanded ? (
+            <FontAwesomeIcon icon={faChevronRight} size="sm" />
+          ) : (
+            <>
+              <FontAwesomeIcon icon={faChevronLeft} size="sm" />
+              <FontAwesomeIcon
+                className="chat-sidebar__controller__second-icon"
+                icon={faCommentDots}
+                size="lg"
+              />
+            </>
+          )}
+        </div>
 
-          <div className="chat-sidebar__tabs">
-            <div className={venueChatTabStyles} onClick={selectVenueChat}>
-              {venueChatTabTitle}
-            </div>
-            <div className={privateChatTabStyles} onClick={selectPrivateChat}>
-              {privateChatTabTitle}
-            </div>
+        <div className="chat-sidebar__tabs">
+          <div className={venueChatTabStyles} onClick={selectVenueChat}>
+            {venueChatTabTitle}
+          </div>
+          <div className={privateChatTabStyles} onClick={selectPrivateChat}>
+            {privateChatTabTitle}
           </div>
         </div>
-        <div className="chat-sidebar__tab-content">
-          {chatSettings.openedChatType === ChatTypes.VENUE_CHAT && (
-            <VenueChat onAvatarClick={setSelectedProfile} />
-          )}
-          {chatSettings.openedChatType === ChatTypes.PRIVATE_CHAT && (
-            <PrivateChats
-              recipientId={chatSettings.recipientId}
-              onAvatarClick={setSelectedProfile}
-            />
-          )}
-        </div>
       </div>
-      <UserProfileModal
-        userProfile={selectedProfile}
-        show={hasSelectedProfile}
-        onHide={unselectProfile}
-      />
-    </>
+      <div className="chat-sidebar__tab-content">
+        {chatSettings.openedChatType === ChatTypes.VENUE_CHAT && <VenueChat />}
+        {chatSettings.openedChatType === ChatTypes.PRIVATE_CHAT && (
+          <PrivateChats
+            recipientId={chatSettings.recipientId}
+            onAvatarClick={openUserProfileModal}
```

Comment:
I think we still should remove `onAvatarClick` as a prop here and use openUserProfile directly in `PrivateChats` component